bindata/cert-manager-deployment/networkpolicies/cert-manager-api-server-egress-networkpolicy.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
These manifests will get removed when [script](https://github.com/openshift/cert-manager-operator/blob/master/hack/update-cert-manager-manifests.sh#L35) gets called. I think instead we can have these in `bindata/networkpolicies` common for cert-manager and istio-csr?